ProjFS.Mac/PrjFSKextTests/KauthHandlerTests.mm

```diff
@@ -357,6 +357,8 @@ - (void)testShouldHandleFileOpEvent {
     shared_ptr<mount> testMountNone = mount::Create("none", fsid_t{}, 0);
     shared_ptr<vnode> testVnodeNone = vnode::Create(testMountNone, "/none");
     shared_ptr<vnode> testVnodeUnsupportedType = vnode::Create(testMount, "/foo", VNON);
+    std::string fromFilePath = repoPath + "/fromFile.txt";
+    std::string fromDirectoryPath = repoPath + "/fromDirectory";
```

Comment:
Nope, those have moved to `HandleFileOpOperationTests.mm` in later iterations instead and have since been further modified; I've removed this leftover change.